### DISCUSSION TITLE
Using tailwindcss for output rendering

### DISCUSSION THREAD

**Discussion by arisawali2014** [awcodes/filament-tiptap-editor#400] [Q&A]
Is there any solutions for use tailwindcss instead for styling?
i want to use it as default but i don't know how

**Comment by awcodes** [comment]
You're not going to be  able to use tailwind directly on the output. If the classes are stored in the db then there's no way for tailwind to know to include them in the build. 

What I do is use the tailwind typography plugin to handle the heavy lifting of the styling then tweak where necessary in the css file for the site.

**Reply by arisawali2014** [reply]
What about grid builder. can we use tailwindcss also to handle this by default?

**Reply by awcodes** [reply]
You can do the css any way you want. If you want to use tailwind classes then you can still do so in a stylesheet with the @apply directive.
